src/config/dynamic-theme-fixes.config

```diff
@@ -26594,6 +26594,16 @@ div[style='background-image: url("https://web-assets.strava.com/assets/react-18/
     background-blend-mode: multiply !important;
     background-color: ${rgba(255,255,255,0.5)} !important;
 }
+.leaflet-control-attribution:after,
+.leaflet-control-zoom-in,
+.leaflet-control-zoom-out,
+.leaflet-popup-close-button,
+.leaflet-control-layers-toggle,
+.leaflet-container.dark .map-tooltip .close,
+.map-tooltip .close,
+.mapbox-icon {
+    background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADQAAAIICAMAAAAWgT0mAAAABGdBTUEAALGPC%2FxhBQAAAAFzUkdCAK7OHOkAAABgUExURQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAPpP6I0AAAAfdFJOUwDYoAgNNvSA4%2Fu3mcTrUb5VQCByjBcoRwJMrs9nfF29SYbmAAAFH0lEQVR42u2c55qrIBCGY8HeYkliC%2Fd%2Fl0cRlRjFgbO7z5b5fiV5fAUGhHEYcrmgUKhfpIDSQBmig%2F44FNAXBeAyRH0epFU97Nz%2FfzRQKBQKhUKhvrnv%2F0t9WBQKhUKhUKhvLdfd%2FyxlKHX3Pp9B85XCRzClwCwXKzH8ckVmolQZTrmXz4d0qqdjCB2T63Su1jDSGrBajwYKhUKhUEoibmo4jpG6BIwkxhqfMhIQkkevYa0oP2eu8TaAFl9PGZ%2B%2ByT%2Bh8pjuKJbXcG3P%2FXK5r%2B2S2u01FLh%2Bk9nQOIIMSZ8K7XipHqXkzGHb07EPkh5DKaRJ2%2BodN8qhR4agzsdCxjFkgAyxgVKQyTeQC%2BrcDUQ%2BdhjpDVitR0PvIdR63PUmFr0pTG%2By1JyWUSgUCgXW46FKVM8mjptnpeBg9xFfpfyoh8zmdhJ6r0uNFya2jLgVqb%2B3fPppcTuEqEQfC%2BVF4OwBTlBI11CrfjdEbQEsuNpDaoFdy5%2FZevanyvtuTax7acis5wTdps15x6xzZnKjvPICrWtpwPvJabo87xoH%2Bwn76Rv3EwqFQqG%2BSnakmJm4pWSXua%2BRrJWCMRtKAr3HKZ9qEGMyHwaJ3xdGAWKM%2F4xUoInJmDWg0MwwGwKhlRkpGCQyAwWC7itztrm9QmPjOXO2jS5Uz4543U7HuHiBfb%2BoQ7Lffhqk9HKnDbmqew3HFOZIoFAo1N8Q6RvPa3qVrTHbnBcLExyOIEK0xAMWZo9MWFdVHY4UrKyhbh7fgbwOvAmq3HD7ZdfyKt%2F8XdQPdVu%2FDTXsAVBDab1%2BqyltANDQDCFGXg0N1IFineq1OoYIP8vk2841NIYR1MMhno5ftD4ahoo3NT6EcRsS7pJp5RR%2Ff0rZtXS13FFM80WhUChQkEA6x2olMn7dbvj7pW2fXJNQDSpY5kKmBHG%2FUgnybQ2I%2BTjPVs0QzKs0FK1nir%2F9BmjjDitDlk5JNx2o%2FtP99Fsg9xgCT7EChG4sCoVC%2FVjZJLudpsVaL4FNqxz3dOPnxXqaknyyOhIoq53PUTi0k5RUUoEq10WnzLJjKKXUu88Framxvp9aspLGtaxaQn3L%2BR1Zilw93ThIHtxH5pJmyVlzYoPTNqYASbOKM2Ls5vhK%2Byqgu5mC8mBsv%2F9GIw%2F72t4u1J0EEfeO%2Ffhn2dW5%2Bd6q06C5G72V5Z%2FmMpL3vGpApDjbUi0oy7J5aZUDTLTMhGbFoIj04yoeZfLOyyGu2Yp180vIvsYtFMzgmNC80aorm9aJjQCWm4pCoVCor9CDlNO51JJAj05ZhbhAFaA5PdmeUwAcnHU18sLMPZfFBJXjt2FHKtKVLaAsfpC6XZfnzjk7SG0xG%2Fj9aOfqnpHRcDk7Zu4d27Bgdx33wztvWakn16w47NOYt7pa%2FnvUsPnZ6viol5mDPKY3NOPivhrOlJ1DL3k9MpZPcY9np5el7JbHvjzbdh%2Fs%2FLywvfrp%2B012tt6YQtcPZ3Impz4b8IcvOfLuTJ7xjZ9VT%2Bcz7Lbs%2FwLiqQ3VdFvLmb1%2FIoMifmePxdevS6Q9k53IN%2FkwG6xlz066ObvrqcyXj8f6jbdt5iY9PJnJp0P77a1mzm4zF9TJ%2F2Sg5e9YrJtIS33zMY%2Fiw2G0ZGvT6eQSGXvLnsZhcTwLeetrDHdF8%2BnlSPJoXBJ%2Ffo2xfaMgpJj9benkUsx1SVSmlpIP0lDpFWV8jb4trYMGVAdT26sdYZPlYMNC%2BFMI4LTM3vN6M4qVFgAU6iv0D5QnfMaZoP9%2FAAAAAElFTkSuQmCC) !important;
```

Comment:
Is there a way to not include the data URL? Could you please share a link? When I open the website, I see the icons on the map are not inverted.